stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -163,6 +164,15 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
                 }
             }
         }
+
+        supportFragmentManager.registerFragmentLifecycleCallbacks(
+            object : FragmentManager.FragmentLifecycleCallbacks() {
+                override fun onFragmentStarted(fm: FragmentManager, fragment: Fragment) {
+                    viewBinding.addButton.isVisible = fragment !is PaymentOptionsListFragment
```

Comment:
Can this be `fragment is PaymentOptionsAddCardFragment`?

Can we also add a test for this?

Reply:
Yes